templates/security/fips.html

```diff
@@ -29,20 +29,45 @@ <h2 class="p-heading--4">FIPS 140-2 validated cryptography for Linux workloads</
-
-    <hr class="p-separator">
+<section class="p-strip--light is-bordered">
+  <div class="row u-equal-height">
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Run regulated workloads</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        U.S Federal agencies and anyone deploying systems and cloud services for Federal government agency use, whether directly or through contractors, are required to run workloads with FIPS 140 validated cryptography. FIPS 140 has also been adopted outside of the public sector in industries where data security is heavily regulated, such as financial services (PCI-DSS), healthcare (HIPAA), and other sectors. <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide FIPS 140 certified cryptographic packages.
+      </p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Reduce your compliance costs</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        Developing applications that comply with FIPS140 can be a challenging task. Validating the used cryptography in-house involves a long and expensive process that requires cryptography expertise and involves reviews from a 3rd party lab and NIST. All these introduce costs and complexity that may delay your launch. Ensure that you ship on time and reduce both validation costs and time by using the Ubuntu validated standard open source packages. The <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> packages are validated on common CPU types and are also available for use on the <a href="/public-cloud">public cloud</a> with Ubuntu Pro FIPS.
+      </p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Get NIST certified compliance</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        FIPS 140 ensures that cryptographic algorithms known to be secure are used for data protection, and they are thoroughly tested and attested by a laboratory accredited under the NIST’s Cryptographic and Security Testing (CST) <a class="p-link--external" href="https://www.nist.gov/nvlap/" />Laboratory Accreditation Program (LAP)</a> in the US and <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program">CCCS’s Cryptographic Module Validation Program (CMVP)</a> in Canada. <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide you with cryptographic packages that are tested and attested by atsec Information Security, a NIST accredited laboratory.
+      </p>
+    </article>
+  </div>
+  <p>
+    <a class="p-button--neutral" href="/security/certifications/docs/fips">Get Ubuntu with the FIPS140 validated modules</a>
```

Comment:
1. These links should be in a `row` or `u-fixed-width` see below: 
![Screenshot 2021-07-13 at 13 36 47](https://user-images.githubusercontent.com/58959073/125452689-6eae20cb-aa60-40eb-b18f-e47339a59030.png)
2. The text should say `Read more about the FIPS certification`